packages/dds/map/src/mapKernel.ts

```diff
@@ -481,7 +481,7 @@ export class MapKernel {
         const previousValue = this.get(key);
         this.data.set(key, value);
         const event: IValueChanged = { key, previousValue };
-        this.eventEmitter.emit("valueChanged", event, local, op, this.eventEmitter);
+        this.eventEmitter.emit("valueChanged", event, local, op ?? null, this.eventEmitter);
```

Comment:
Note that the typings suggest `null`, but we were actually possibly emitting `undefined` as per this local typing. Not sure which is really correct, but generally we lean towards `undefined`.